server/routes/place/html.py

```diff
@@ -81,8 +82,7 @@ def place(place_dcid=None):
   else:
     place_name = place_dcid
 
-  place_summary = current_app.config['PLACE_EXPLORER_SUMMARIES'].get(
-      place_dcid, {'summary': ''})
+  place_summary = place_summaries.get_place_summaries().get(place_dcid, "")
```

Comment:
can you add a logging to time this function call?

Reply:
Added logging!